.castor/docker.php

```diff
@@ -295,6 +295,107 @@ function workers_stop(): void
     stop(profiles: ['worker']);
 }
 
+#[AsTask(description: 'Push images cache to the registry', namespace: 'docker', name: 'push', aliases: ['push'])]
+function push(): void
+{
+    // Generate bake file
+    $composeFile = variable('docker_compose_files');
+    $composeFile[] = 'docker-compose.builder.yml';
+
+    $targets = [];
+
+    /** @var string|null $registry */
+    $registry = variable('registry');
+
+    if ($registry === null || $registry === '') {
+        throw new \RuntimeException('You must define a registry to push images.');
+    }
+
+    foreach ($composeFile as $file) {
+        $path = variable('root_dir') . '/infrastructure/docker/' . $file;
```

Comment:
Instead of this, you should use 

```php
$services = json_decode(
    docker_compose(
        ['config', '--format', 'json'],
        context()->withQuiet(),
        profiles: ['*'],
    )->getOutput(),
    true,
)['services'];    
```